### PR TITLE
Add linux support

### PR DESCRIPTION
**Pull request by mzch**
Add linux support

### CODE REVIEW

File: Mod_Lang_CHT/Mod_Lang_CHT.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public string Name
                             dst_path = "Cities.app/Contents/Resources/Files/Locale/" + locale_name + ".locale";
                             break;
                         case Platform.Linux:
-                            //TODO: find locale path under linux
+                            dst_path = "Files/Locale" + locale_name + ".locale";
```
Comment (ccpz) [inline Mod_Lang_CHT/Mod_Lang_CHT.cs]: Do we need to append "/" before local_name?
like:

```
"Files/Locale/" + locale_name + ".locale";
```

**Comment by letlive** [comment]
Oh, sorry, yes, "/" is needed.

**Comment by mzch** [comment]
Ah, sorry, yes, "/" is needed.

**Comment by ccpz** [comment]
merged, thanks :)

**Pull request merged**